app/view_models/journey_navigation.rb

```diff
@@ -0,0 +1,83 @@
+class JourneyNavigation
+  extend Forwardable
+
+  Step = Class.new do
+    extend Forwardable
+
+    def_delegators :@guide, :slug, :url, :label, :concise_label
+
+    attr_accessor :current, :steps
+
+    def initialize(guide, current = false, steps = [])
+      @guide = guide
+      self.current = current
+      self.steps = steps
+    end
+
+    def current?
+      current
+    end
+
+    def steps?
+      !steps.empty?
+    end
+  end
+
+  attr_accessor :journey_tree, :current_step
+
+  def initialize(journey_tree, current_step = nil)
+    self.journey_tree = journey_tree
+    self.current_step = current_step
+  end
+
+  def active?
```

Comment:
This needs tests, and should be memoized. BUT, the other option is moving this responsibility out of this class, perhaps get the controller to ask something for the active journey, and then the view can just check if there is one (rather than if one is active).